pkgs/build-support/release/source-tarball.nix

```diff
@@ -117,7 +117,7 @@ stdenv.mkDerivation (
       version = version + versionSuffix;
     };
 
-    meta = (if args ? meta then args.meta else {}) // {
+    meta = (lib.optionalAttrs (args ? meta) args.meta) // {
```

Comment:
`lib` isn't passed as an argument, causing the build to fail:
```
error: undefined variable 'lib'

       at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-1/pkgs/build-support/release/source-tarball.nix:120:13:

          119|
          120|     meta = (lib.optionalAttrs (args ? meta) args.meta) // {
             |             ^
          121|       description = "Source distribution";

       … while calling 'sourceTarball'
```

Reply:
I removed my changes to this file. No idea how to import lib here.

Reply:
You add it here: https://github.com/NixOS/nixpkgs/blob/4f1b2d513327b121cbf82b4086b3b854a48347a3/pkgs/build-support/release/default.nix#L8

Reply:
I tried that before. Have a look at the error.

Reply:
Try replacing the reference to `args` in this line with `(builtins.removeAttrs args [ "lib" ])`: https://github.com/NixOS/nixpkgs/blob/63b5955814db30d2e2ff7157aaa5665b502ed2f4/pkgs/build-support/release/source-tarball.nix#L76